### ISSUE TITLE
Additional cluster data broke sharded cluster

### ISSUE THREAD

**Issue by gcoupelant** [mgmonteleone/python-atlasapi#177]
Hello

In this MR: https://github.com/mgmonteleone/python-atlasapi/pull/157 you added a `created_date` attribute to the `ClusterConfig` class and updated the `fill_from_dict` class method accodingly.
But you omitted to also update the `ShardedClusterConfig` class.

Because of that, some commands will fail when ran against a sharded cluster:
```
TypeError: ShardedClusterConfig.__init__() takes from 1 to 21 positional arguments but 22 were given
```

The 22nd positional argument being `created_date`.

Downgrading to version 2.0.6 "fixes" the issue for now 🙂

**Comment by mgmonteleone** [comment]
Thanks for letting me know, fixing now.

**Comment by mgmonteleone** [comment]
@gcoupelant , fixed and released in V3.0.1b5.
